medinify/scrapers/webmd_scraper.py

```diff
@@ -6,7 +6,7 @@
 """
 
 import re
-import requests
+#import requests
```

Comment:
Why was this change made?

Reply:
Change was made only for my own testing since I do not have that package on my personal laptop. I completely forgot to change it back, since it was a change I made last Friday, sorry!